src/fitnesse/testsystems/slim/tables/QueryTable.java

```diff
@@ -113,7 +113,7 @@ protected ExecutionResult markRows(QueryResults queryResults, Iterable<MatchedRe
     markMissingRows(unmatchedTableRows);
     markSurplusRows(queryResults, unmatchedResultRows);
 
-    return unmatchedTableRows.size() > 0 || unmatchedResultRows.size() > 0 ? ExecutionResult.FAIL : ExecutionResult.PASS;
+    return !unmatchedTableRows.isEmpty() || !unmatchedResultRows.isEmpty() ? ExecutionResult.FAIL : ExecutionResult.PASS;
```

Comment:
I considered flipping this ternary to (isEmpty && isEmpty) ? PASS : FAIL, but I'm not sure whether that makes it more readable or less.

(Same thing with OrderedQueryTable above.)